go/types/encode_human_readable_test.go

```diff
@@ -406,9 +406,9 @@ func TestEmptyCollections(t *testing.T) {
 	c := MakeMapType(BlobType, NumberType)
 	assertWriteTaggedHRSEqual(t, "Type(Map<Blob, Number>)", c)
 	d := NewMap()
-	assertWriteTaggedHRSEqual(t, "Map<, >({})", d)
+	assertWriteTaggedHRSEqual(t, "Map<, >", d)
```

Comment:
Oh I misunderstood the issue. So you're saying it was the spacing of the types that was the problem?